case/src/test/kotlin/com/ritense/case/service/CaseDefinitionServiceTest.kt

```diff
@@ -85,4 +96,118 @@ class CaseDefinitionServiceTest {
             val foundCaseDefinitionSettings = service.updateCaseSettings(caseDefinitionName, caseSettingsDto)
         }
     }
+
```

Comment:
You should probably also test the happy path

Reply:
the happy path is tested in the integration test